README.md

```diff
@@ -135,6 +135,16 @@ git, I worked on github and gitlab. I tried using git guis but I prefer to work
 
 * Explain event delegation
 * Explain how `this` works in JavaScript
+    This points to an object depending on the context,
+    In what function was used in which we defined it.
+    Context, ie the way and place of the call.
+
+    function ab() { this } - window (because window.ab)
+    var obj = { fun: ab }; obj.fun(); - obj
+    var obj = { fun: function() { ab(); }} - window
+    ab.call(obj) - obj
+    const a = ab.bind(obj) - obj
```

Comment:
Bazowałem na tym:
http://ferrante.pl/frontend/javascript/this-w-javascript/

Przykłady poprawiłem na bazie Twojego linku.